guided-trials-default-custom-user-data/default-custom-user-data.gradle

```diff
@@ -135,6 +135,23 @@ void addCiMetadata() {
             buildScan.value 'CI agent ID', agentId
         }
     }
+
+    if (isGitHubActions()) {
+        def repo = System.getenv('GITHUB_REPOSITORY')
+        def runId = System.getenv('GITHUB_RUN_ID')
+        if (repo && runId) {
+            buildScan.link 'GitHub Actions run', "https://github.com/$repo/actions/runs/$runId"
+        }
+        if (System.getenv('GITHUB_HEAD_REF')) {
```

Comment:
wrong copy paste ?
should be "if GITHUB_WORKFLOW"

Reply:
🤦‍♂

Reply:
Fixed.